longevity_lwt_test.py

```diff
@@ -0,0 +1,87 @@
+import re
+
+from longevity_test import LongevityTest
+from sdcm.sct_events import LwtFailureEvent
+
+
+class LWTLongevityTest(LongevityTest):
+
+    def test_lwt_longevity(self):
+        self.test_custom_time()
+        self.validate_data()
+
+    def validate_data(self):
+        node = self.db_cluster.nodes[0]
+        with self.cql_connection_patient(node, keyspace='cqlstress_lwt_example') as session:
+            self.validate_not_updated_data(session=session)
+            self.validate_updated_data(session=session)
+
+    def validate_not_updated_data(self, session):
+        """
+        User profile 'c-s_lwt_data_not_for_update.yaml' prepares data for "blogposts" table that used in the LWT
+        longevity. This data won't be updated during the test because of values for partition keys are higher then max
+        PK value of the test:
+        - data_dir/c-s_lwt_basic.yaml primary key column definition:
+            population: seq(1..10000000)
+        - data_dir/c-s_lwt_data_not_for_update.yaml primary key column definition:
+            population: seq(30000000..30000100)
+
+        In case the 'c-s_lwt_data_not_for_update.yaml' user profile exists in 'prepare_write_cmd', expected that the
+        rows data with primary keys "30000000 - 30000100" won't be updated.
+
+        This function verifies that
+        """
+        prepare_write_cmd = self.params.get('prepare_write_cmd', default=[])
+
+        cmd_not_for_update = [cmd for cmd in prepare_write_cmd if 'c-s_lwt_data_not_for_update.yaml' in cmd]
+
+        if cmd_not_for_update:
+            self.log.info('Verify that rows, that shouldn\'t be updated, were not updated')
+            n_param_search = re.search('n=([0-9]*)', cmd_not_for_update[0])
+            if n_param_search:
+                min_not_updated_partition_key = 30000000
+                max_not_updated_partition_key = min_not_updated_partition_key + int(n_param_search.group(1))
+                not_updated_pks = ','.join([str(i) for i in range(min_not_updated_partition_key,
+                                                                  max_not_updated_partition_key)])
+                # Disable paging
+                session.default_fetch_size = 0
```

Comment:
why?

Reply:
If we will choose to use bigger number than 100 will it work ok in the timeout bundries?

Reply:
the logic of the function was changed